source/extensions/tracers/xray/tracer.cc

```diff
@@ -0,0 +1,122 @@
+#include "extensions/tracers/xray/tracer.h"
+
+#include <random>
+
+#include "common/common/utility.h"
+#include "common/tracing/http_tracer_impl.h"
+
+#include "extensions/tracers/xray/util.h"
+#include "extensions/tracers/xray/xray_core_constants.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace XRay {
+
+std::string Tracer::generateRandom96BitString() {
+  char values[25] = {'\0'};
+  uint64_t seed = std::chrono::duration_cast<std::chrono::nanoseconds>(
+                      time_source_.systemTime().time_since_epoch())
+                      .count();
+  std::mt19937 mt_rand(seed);
+
+  for (int i = 0; i < 24; i++) {
+    values[i] = hex_digits[mt_rand() % 16];
+  }
+  return values;
+}
+
+std::string Tracer::generateTraceId() {
+  uint64_t epoch_time =
+      std::chrono::duration_cast<std::chrono::seconds>(time_source_.systemTime().time_since_epoch())
+          .count();
```

Comment:
The reason I need this is because of the concept of `TraceID` which contains three parts:
1. The version number, that is, 1.
2. The time of the original request, in Unix epoch time, in 8 hexadecimal digits. For example, 10:00AM December 1st, 2016 PST in epoch time is 1480615200 seconds, or 58406520 in hexadecimal digits.
3. A 96-bit identifier for the trace, globally unique, in 24 hexadecimal digits.

In order to meet the second requirement, I need to get the epoch_time and then translate into hex type.